src/Poloniex/Spot/API/OrderBook.jl

```diff
@@ -0,0 +1,103 @@
+module OrderBook
+
+export OrderBookQuery,
+    OrderBookData,
+    order_book
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Poloniex
+using CryptoAPIs: Maybe, APIsRequest
+
+@enum OrderBookLimit begin
+    FIVE = 5
+    TEN = 10
+    TWENTY = 20
+    FIFTY = 50
+    HUNDRED = 100
+    HUNDRED_FIFTY = 150
+end
+
+Base.@kwdef struct OrderBookQuery <: PoloniexPublicQuery
+    symbol::Maybe{String} = nothing
+    limit::Maybe{OrderBookLimit} = nothing
+    scale::Maybe{Int64} = nothing
+end
+
+function Serde.SerQuery.ser_type(::Type{OrderBookQuery}, x::OrderBookLimit)::Int64
+    return Int64(x)
+end
```

Comment:
Is it necessary?

Reply:
yes, it is necessary the query requires a numeric value for the field `limit`